README.md

```diff
@@ -25,15 +25,9 @@ animal.name === 'jon'
 
 ## API
 
-### merge(destination, source)
+### merge(Object destination, Object source [, bool overwrite])
 
-Redefines `destination`'s descriptors with `source`'s. The return value is the
-`destination` object.
-
-### merge(destination, source, false)
-
-Defines `source`'s descriptors on `destination` if `destination` does not have
-a descriptor by the same name. The return value is the `destination` object.
+Merge properties from `source` object into `destination` object, preserving each property's descriptors.  If `overwrite == false` then existing properties on `destination` will not be overwritten.  The default value of `overwrite` is true.  Returns the destination object.
```

Comment:
Please hard wrap the lint to approx 80 chars. And use a single space instead of double space.